packages/url-loader/src/plugins/overlays.ts

```diff
@@ -38,42 +112,21 @@ export function plugin(props: PluginSettings) {
 
   if ( typeof text === 'string' ) {
     applyOverlay({
-      text: {
-        ...DEFAULT_TEXT_OPTIONS,
+      text: Object.assign({}, DEFAULT_TEXT_OPTIONS, {
```

Comment:
suggestion: probably not now but maybe in the future you could use defu for such cases?https://dev.to/jacobandrewsky/assigning-default-properties-efficiently-with-defu-2in2

Reply:
took a quick look and definitely not opposed to it! i'll read more about it to understand what exactly it does. want to make an Issue for it?

Reply:
Sure, added!

https://github.com/colbyfayock/cloudinary-util/issues/126